game/game/gamescript.cpp

```diff
@@ -404,10 +404,15 @@ void GameScript::initDialogs() {
 
 void GameScript::loadDialogOU() {
   auto gCutscene = Gothic::inst().nestedPath({u"_work",u"Data",u"Scripts",u"content",u"CUTSCENE"},Dir::FT_Dir);
-  static const char* names[] = {
-    "OU.DAT",
-    "OU.BIN",
-    };
+  std::string prefix = std::string(Gothic::inst().settingsGetS("FILES","OutputUnits"));
+  std::vector<std::string> names;
```

Comment:
I suppose this is suboptimal compared to the previous state, please feel free to change to your taste.

Reply:
This part is OK - one more allocation wont hurt load-time